lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp

```diff
@@ -211,6 +217,12 @@ class LowerToBackendContractPass
   void runOnOperation() override {
     ModuleOp module = getOperation();
 
+    if (decompose) {
+      MLIRContext *context = &getContext();
+      ConversionTarget target(*context);
+      markDecomposedOpsAsIllegal(context, target, backendLegalOps);
```

Comment:
Is there something missing here? I would expect `target` to be used somewhere. In particular in satisfiesBackendContract.

(also is it possible to test this? this does sound like something that a .mlir test would be useful for)

Reply:
Whoops! Not sure why my brain thought all I had to do was set the target. Should be fixed now